hpcgap/lib/fldabnum.gi

```diff
@@ -1646,101 +1646,6 @@ InstallMethod( Coefficients,
     end );
 
 
-#T ##########################################################################
-#T ##
-#T #F  NumberRing( gens )  . . . . . . . . . . . integral ring in a number field
```

Comment:
First off, it was removed here in this PR because it was previously removed in `lib/fldabnum.gi`.

It was removed there because it was commented out since August 1996 and nobody seems to have missed it in the past 22 years... :)

Reply:
Ah right! :)